src/braket/aws/aws_quantum_job.py

```diff
@@ -84,8 +85,11 @@ def create(
         """Creates a job by invoking the Braket CreateJob API.
 
         Args:
-            device (str): ARN for the AWS device which is primarily
-                accessed for the execution of this job.
+            device (str): ARN for the AWS device which is primarily accessed for the execution
+                of this job. Alternatively, a string of the format
+                "local:<provider>.<simulator>.<name>" for using a local simulator for the job.
```

Comment:
Without being able to be more specific for the "provider", "simulator", and "name", I'm not sure what use we get out of having the format. It seems like we'll just confuse customers and (since it's not standardized) customers may just send in "a.a.a". 

What about making this the same as our LocalSimulator "name" parameter:
The name of the simulator backend or
                the actual simulator instance to use for simulation. Defaults to the
                `default` simulator backend name.

Reply:
Or similar, if not the same.

Reply:
I see you updated the check - we should also update the docs.

Reply:
Format and docs updated to match the decision to use `"local:provider/simulator"` convention